doc/making_releases.md

```diff
@@ -12,6 +12,14 @@ NOTE: it is untested on Windows
 After the script runs it places the created releases in the `builds`
 folder.
 
+## Installing export templates
+Godot's export templates must be installed to export the project. Instructions
+can be found here: 
+https://docs.godotengine.org/en/stable/tutorials/export/exporting_projects.html?highlight=export%20templates
+
+Alternatively, you can run a script in this repository to do this. Use the
+command `dotnet run --project Scripts -- godot-templates`.
```

Comment:
This apparently only works on Linux though, maybe that should be noted?

Reply:
I wouldn't put it in this document, because it is more likely that the script is updated to work on more platforms sooner than this document is edited again.

Reply:
And that might actually push someone to make sure it works on Windows as well.